lsm303agr/lsm303agr.go

```diff
@@ -0,0 +1,195 @@
+/*
```

Comment:
Could you please make all of the comments follow the standard Go style, that also makes them work with GoDoc?

https://blog.golang.org/godoc

For example, the following comment block should be exactly like this instead:

```
// package lsm303agr implements a driver for the LSM303AGR,
// a 3 axis accelerometer/magnetic sensor which is included on BBC micro:bits v1.5.
//
// Datasheet: https://www.st.com/resource/en/datasheet/lsm303agr.pdf
```